sdcard/rootfs/kube-systemd/etc/kubernetes/dropins/docker-flannel.conf

```diff
@@ -18,5 +19,4 @@ ExecStart=/usr/bin/docker daemon \
     --exec-opt native.cgroupdriver=cgroupfs
 Restart=on-failure
 RestartSec=2
-MountFlags=
```

Comment:
Ah, this should be here too. Sorry if confusing.
The first statement resets the `MountFlags` from `docker.service`, the other one sets the value.
The same method is applied to `ExecStart`